doc/rvfi.rst

```diff
@@ -0,0 +1,27 @@
+.. _rvfi:
+
+RISC-V Formal Interface
+=======================
+
+Ibex is adapted to support the interface defined by RISC-V Formal Interface
+(riscv-formal_).  The signals provide an insight to the state of the core.
+Generally speaking the interface decodes the current instruction.
+This includes the opcode, the source and destination registers,
+the program counter and for memory operations the memory location and data.
+
+
+Formal Verification
+-------------------
+
+The signals provided by RVFI can be used to formally verify the adherents of
+Ibex to the RISC-V specification_.
+Currently the implementation is restricted to support the `I` and `C` extensions.
+
+At the moment Ibex is not formally verified, a previous version (zero-riscy)
+was tested, but this required changes to the core as well as to the tool used
+in the process (yosys_). Support is work in progress.
+
+
+.. _riscv-formal: https://github.com/SymbioticEDA/riscv-formal/blob/master/docs/rvfi.md
+.. _specification: https://riscv.org/specifications/
+.. _yosys: https://github.com/YosysHQ/yosys
```

Comment:
It's up to you, but if you prefer in line links you can do that as well in rst.

```
... the `RISC-V specification <https://riscv.org/specifications/>`_.
```

Reply:
Normally I do not do it inline and I saw no an example how it is done in this documentation, but I will follow convention and do it inline.

Reply:
It's really up to you, no strict convention or requirement either way.